src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,67 @@
+package duke.task;
+import java.util.List;
+import java.util.ArrayList;
+/**
+ * Abstract class Task for Todo, Deadline, and Event to inherit from
+ *
+ * @author Pearl Twe
+ * @version CS2103T AY22/23 Semester 2
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Return description of task
+     * @return description
+     */
```

Comment:
The verb should be in plural form, and there should be space between the description and @return line